lib/gnostic.go

```diff
@@ -37,6 +37,21 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+// UsageError is a response to invalid command-line inputs
+// and should trigger display of usage (help) information.
+type UsageError struct {
+	Message string
+}
+
+func (e *UsageError) Error() string {
+	return fmt.Sprintf("%s", e.Message)
```

Comment:
why do `fmt.Sprintf` here? Can it be just `return e .Message`?

Reply:
yes, fixed.